ang/crmCaseType.js

```diff
@@ -230,6 +249,12 @@
       }
     };
 
+    /// Clears the activity's default assignee values for relationship and contact
```

Comment:
I wonder why `///` is used here instead of `//`?

Reply:
That's the convention used for function docs